Dockerfile

```diff
@@ -16,11 +16,13 @@ RUN pip wheel . --wheel-dir=/wheels
 # Install from wheels
 FROM ghcr.io/apeworx/ape:${BASE_APE_IMAGE_TAG:-latest}
 USER root
-COPY --from=builder /wheels /wheels
+COPY --from=builder /wheels/*.whl /wheels
 RUN pip install --upgrade pip \
-    && pip install silverback \
+    && pip install \
+    --no-cache-dir --no-index --find-links=/wheels \
     'taskiq-sqs>=0.0.11' \
```

Comment:
don't we need `taskiq-redis` as well?

Reply:
I was told we do not by @mikeshultz, please confirm

Reply:
I would think we need it for the results backend

Either way, it also helps for testing to have it, since you can just run the redis container alongside it (SQS is not as easy to set up)

Reply:
what pin do we want?

Reply:
> I was told we do not by @mikeshultz, please confirm

Context? It is needed for the results backend.

Reply:
In the Dockerfile, we were adding installs in this location, I added taskiq-sqs and a few others, but I remember being told we don't need the others. I'm likely having a phantom memory here